### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pankhurisaxena28**

### CODE REVIEW

**Comment by sethvargo** [comment]
Hi @pankhurisaxena28 - the docs are auto-generated from the values in `action.yml`. If you want to change these, please update `action.yml` and then run `npm run docs` or trigger a release (which will re-compile the docs automatically).

File: README.md [hunk 1]
```diff
@@ -96,28 +93,27 @@ jobs:
...
     is evaluated against the count of issues with that severity in the IaC
     scan results and then severity level evaluations are aggregated using
-    `AND` or `OR` to arrive at `failure_criteria` value.
+    `AND` or `OR` to arrive at `failure_criteria` value. You must include an aggregator in the string. The aggregator value is case-sensitive.
+
+    For example, if you set the failure criteria to `HIGH:1,LOW:1,OPERATOR:OR`, the workflow fails if there is 1 or more HIGH severity findings <strong>or</strong> 1 or more LOW severity findings. If you set the failure criteria to `HIGH:1,LOW:1,OPERATOR:AND`, the workflow fails if there is 1 or more HIGH severity findings <strong>and</strong> 1 or more LOW severity findings.
 
     If the `failure_criteria` evaluates to `true`, the workflow is marked as
-    `FAILED`. Otherwise, the workflow is marked as `SUCCESS`.
+    `FAILED`. Otherwise, the workflow is marked as `SUCCESS`. The default is 'Critical:1,High:1,Medium:1,Low:1,Operator:OR'.
```
Comment (sethvargo) [inline README.md:116]: These are auto-generated from https://github.com/google-github-actions/analyze-code-security-scc/blob/main/action.yml

Comment (pankhurisaxena28) [reply README.md]: Updated the PR as per your suggestions.

File: action.yml [hunk 2]
```diff
@@ -38,40 +38,48 @@ inputs:
 
   iac_type:
     description: |-
-      IaC template type. Currently only `terraform` is supported.
```
Comment (sethvargo) [inline action.yml:41]: You can remove "the default is" sentences from all of these. The generated documentation will automatically insert that.

Comment (pankhurisaxena28) [reply action.yml:41]: Done

**Review by sethvargo — COMMENTED** [review]
A few nits. It might help to configure your editor to remove trailing whitespace on save.

**Review by sethvargo — APPROVED** [review]

**Pull request merged**